frameworks/elastic/universe-kibana/resource.json

```diff
@@ -1,12 +1,12 @@
-{
-  "assets": {
-    "uris": {
-      "kibana-tar-gz": "https://artifacts.elastic.co/downloads/kibana/kibana-{{elastic-version}}-linux-x86_64.tar.gz"
-         }
-  },
-  "images": {
-    "icon-small": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg",
-    "icon-medium": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg",
-    "icon-large": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg"
-  }
+{  
```

Comment:
There should be no trailing whitespace in the file: Use the following contents:
```json
{
  "assets": {
    "uris": {
      "kibana-tar-gz": "https://artifacts.elastic.co/downloads/kibana/kibana-{{elastic-version}}-linux-x86_64.tar.gz"
    }
  },
  "images": {
    "icon-small": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg",
    "icon-medium": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg",
    "icon-large": "https://static-www.elastic.co/assets/blt282ae2420e32fc38/icon-kibana-bb.svg"
  }
}
```
(The only difference is the removed line and trailing comma)